typings/index.d.ts

```diff
@@ -44,6 +44,40 @@ declare namespace CodeceptJS {
     Scenario: number;
   };
 
+  type AiPrompt = {
+    role: string;
+    content: string;
+  }
+
+  type AiConfig = {
+    /** request function to send prompts to AI provider */
+    request: () => Promise<string>,
```

Comment:
```suggestion
    request: (messages: any) => Promise<string>,
```

Reply:
it's working with my fixing, appreciate your effort and contribution, Davert.
<img width="445" alt="image" src="https://github.com/codeceptjs/CodeceptJS/assets/96564753/800188ea-d5b3-4924-b494-ad56820dce9d">